spec.md

```diff
@@ -469,49 +468,60 @@ Outputs:
 
 Procedure:
 
-1. s' = H(PRF(8 \* ceil(log2(r)))) mod r
+1. (Ch[1],...,Ch[U]) = CGens
 
-2. if subgroup\_check(h0) is INVALID abort
+2. s' = H(PRF(8 \* ceil(log2(r)))) mod r
```

Comment:
Probably should define `ceil` in terminology also even though it probably obvious to most